### PR TITLE
complete homepage

### PR DESCRIPTION
**Pull request by SegunCodes**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,43 +67,22 @@
             <div class="container custom-container">
                 <div class="main-header-bg">
                     <div class="row align-items-center justify-content-between">
-                        <div class="col-xl-3 col-lg-3 col-md-6 col-6">
+                        <div class="col-xl-2 col-lg-3 col-md-6 col-6">
                             <div class="logo">
-                                <a class="logo-img" href="index.html"><img src="assets/img/logo/header_logo_one.png" alt=""></a>
```
Comment (davidfrank96) [inline index.html:77]: Let's Allow this on the page, I have their logo and I'm already working  on their logos, so lets allow this, revert this back to the default

File: index.html [hunk 2]
```diff
@@ -121,7 +95,7 @@
                             </div>
                             <div class="main-header-right-one d-none d-xl-block d-xl-flex align-items-center ">
                                 <div class="donate">
-                                    <a href="#" class="theme_btn">Be a Caregiver <i class="fal fa-long-arrow-right"></i></a>
+                                    <a href="#" class="theme_btn">Request for a service <i class="fal fa-long-arrow-right"></i></a>
```
Comment (davidfrank96) [inline index.html:98]: Don't you think the Request for a service is too long? lets leave it at Be  a caregiver

**Review by davidfrank96 — CHANGES_REQUESTED** [review]
Lets  work on the changes or you leave the homepage for me and focus on the backend first, you know we dont really have time again

File: index.html (hunk 2 above)
Comment (SegunCodes) [reply index.html:98]: Looking at the file sent, `Become a caregiver` and `Request for a service` are different

File: index.html (hunk 1 above)
Comment (SegunCodes) [reply index.html:77]: Okay, no wahala

**Pull request merged**